velox/experimental/cudf/exec/ExpressionEvaluator.cpp

```diff
@@ -200,7 +200,7 @@ std::vector<cudf::ast::literal> createLiteralsFromArray(
 } // namespace
 
 using Op = cudf::ast::ast_operator;
-const std::map<std::string, Op> binaryOps = {
+const std::map<std::string, Op> prestoBinaryOps = {
```

Comment:
Let's use `unordered_map` for `prestoBinaryOps` and `sparkBinaryOps`.
```suggestion
const std::unordered_map<std::string, Op> prestoBinaryOps = {
```